### PR TITLE
fix relocatability

### PR DESCRIPTION
**Pull request by bjarthur**
needed these changes to move a PackageCompiler app from ~one ubuntu~ an oracle linux 9.2 box to ~another~ an ubuntu 22.04 box

### CODE REVIEW

**Comment by bjarthur** [comment]
doh, nevermind, i had forgotten to rebase master.  this PR is ready for review!

**Comment by bjarthur** [comment]
so certain assets, like the one below, are still not relocatable with this PR:

```
module Smiley

using Bonito, RelocatableFolders

function julia_main()::Cint
    display(smiley())
    return 0
end

function smiley()
    App() do
        DOM.div(DOM.img(src=Asset(@path joinpath(dirname(@__DIR__),"smiley.png"))))
    end
end

end
```

File: src/interactive.jl [hunk 1]
```diff
@@ -33,8 +33,8 @@ routes, task, server = interactive_server(Website.asset_paths()) do
     )
 end
 
-# Once everything looks goo, export the static site
-dir = joinpath(@__DIR__, "docs")
+# Once everything looks good, export the static site
+dir = @path joinpath(@__DIR__, "docs")
```
Comment (SimonDanisch) [inline src/interactive.jl:37]: I think this isn't needed?

Comment (bjarthur) [reply src/interactive.jl:37]: oh, perhaps not, if it's just for documentation.  i just simply `grep`'ed for all instances of `__DIR__` and `__FILE__ `and prefixed with `@path`.  i can delete if you like.  but the bigger for me is why does the Smiley MWE above not work?  thanks for the review!

**Pull request closed without merging**